### ISSUE TITLE
training for specific letters

### ISSUE THREAD

**Issue by pkozlows** [shervinemami/practice_speechrec_mappings#4]
Hi, it's me again. I'm using your program to find a talon of alph that works for me. I was wondering whether it would be simple to implement a way to do training just for specific letters. This could be helpful if someone wanted to hone their alphabet, as I am doing currently.

**Comment by shervinemami** [comment]
Yes you can do this without modifying the code of this tool, but it takes some manual steps. Use the Dragonfly method (ie: don't pass the --talon flag), so it uses the alphabet and symbol files included in this project. Then just modify them manually to use whichever phrases you want. For example, if you only want to practice "air" for "a" and "beta" for "b" but not any other letters or numbers or symbols at all, then modify the included alphabet python file so it only has a and b rows.

**Comment by shervinemami** [comment]
(I'm talking about "lettermap.py". It should be obvious how to modify it even to someone that hasn't done any programming)

**Comment by pkozlows** [comment]
Thanks, makes sense.
